_includes/outreach-partners.html

```diff
@@ -0,0 +1,11 @@
+<div id="outreach-partners" class="page-wrapper sponsor-section">
+    <div class="page-details">
+        <div class="title">Outreach Partners</div>
+        <section class="outreach-grayscale">
+            <div class="row">
+                <div class="col s4"><a href="https://fossunited.org" target="_blank"><img class="responsive-img"
+                            src="assets//images/partners/foss-united.png" alt="FOSS United"></div></a>
```

Comment:
Add the src in the following format: `src={{ "assets/images/c/partners/foss-united.png" | relative_url }}`

Also, end img tag with `/>` instead of `>`

Reply:
Done.